test/views/msb/expected_throughputSpec.scala

```diff
@@ -18,7 +18,7 @@ package views.msb
 
 import forms.{Form2, InvalidForm, ValidForm}
 import jto.validation.{Path, ValidationError}
-import models.moneyservicebusiness.ExpectedThroughput
+import models.moneyservicebusiness.{CETransactionsInNext12Months, ExpectedThroughput}
```

Comment:
Do you need this CETransactions import in this Spec?

Reply:
Removed, thanks